benchmarks/src/main/scala/bloop/logging/BloopLoggerBenchmark.scala

```diff
@@ -0,0 +1,40 @@
+package bloop.logging
+
+import java.io.OutputStream
+
+import org.openjdk.jmh.annotations.Benchmark
+
+object BloopLoggerBenchmark {
+  val logger = BloopLogger("benchmark")
+  private val devnull: OutputStream = _ => ()
+  BloopLogger.update(logger, devnull)
+}
+
```

Comment:
Is this benchmark valuable at all? It's only testing the cost of registering one log.

Reply:
Yeah, but given how complicated our setup with logging is (adding and removing the appenders etc.), I thought it was valuable to have it. There are other configuration options in Log4J that can have an impact (for instance, the layout of the log messages). I'd prefer to know what happens when I change them.

Do you want to get rid of that benchmark?